src/types.ts

```diff
@@ -254,6 +254,14 @@ export type IsoDateTime = Branded<string, 'IsoDateTime'>
  */
 export type MonthId = string
 
+/**
+ * Identifies IANA timezone name.
+ * Branded type.
+ *
+ * @example 'America/New_York'
+ */
+export type IanaTimezone = Branded<string, 'IanaTimezone'>
```

Comment:
Wasn't sure if I should add `asIanaTimezone` casting function and any tests so let me know if I should

Reply:
I'm ok without it 👍